src/os/ui/im/importmanager.js

```diff
@@ -32,6 +32,12 @@ os.ui.im.ImportManager = function() {
    */
   this.importUIs_ = {};
 
+  /**
+   * @type {Object<string, Object>}
+   * @private
+   */
+  this.serverTypes_ = {};
```

Comment:
When you have a well-defined object type, it's preferred to create a `@typedef` in externs (`os.externs.js`). This allows the compiler to fully type check the object, and will prevent the compiler from renaming properties on the object which can cause the compiled app to behave differently.

I would recommend creating a typedef called `osx.import.ServerType` and using it here in place of `Object`. Also update the type in any functions that reference this map.